indico/htdocs/js/indico/widgets/protection_widget.js

```diff
@@ -29,16 +34,16 @@
         }, options);
 
         var inputs = $('input[name=' + options.fieldName + '][id^=' + options.fieldId + ']');
+        var $enableAclLink = $('.enable-acl-link');
```

Comment:
I don't like non-uppercase acronyms but it looks pretty bad with all-uppercase too so I usually go for Html/Acl/etc even though it looks like :hankey: